### DISCUSSION TITLE
how to do DOMTY

### DISCUSSION THREAD

**Discussion by Deepakrocknow** [community/community#118408] [Discussions]
### Select Topic Area

Question

### Body

re #1

**Comment by Deepakrocknow** [comment]
re #2

**Comment by LiteBrite82** [comment]
Hi @Deepakrocknow 

Can you provide some more context for your post? What is DOMTY? What is the context of the two links you provided?

**Comment by LiteBrite82** [comment]
Hi @Deepakrocknow ,

Thanks for joining us for the GitHub Universe celebrations in the GitHub Community, we're glad you're here!

If you're looking for help with a general GitHub feature you might want to try asking in the main [GitHub Community](https://github.com/orgs/community/discussions/).

We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
